src/core/posix.cc

```diff
@@ -122,16 +122,16 @@ posix_thread::posix_thread(attr a, std::function<void ()> func)
     }
 #endif
 
-    if (a._affinity) {
```

Comment:
>  core/posix.cc: pthread_attr_setaffinity_np is glibc specific and not portable

this reads like a part of bug report. so i think the title of this commit message can be improve further. ideally, we should use the imperative mood in the title. in this case, we could use something like:

> posix: use pthread_setaffinity_np instead of pthread_attr_setaffinity_np
>
> pthread_attr_setaffinity_np is glibc specific and not portable. without which, 
> we cannot set the affinity when creating a pthread.but without another
> extension pthread_setaffinity_np, we can set the CPU affinity after the thread
> is created. hence this change. since both glibc and musl libc implements this
> non-portable extension, we can use it here as an alternative of 
> pthread_attr_setaffinity_np.

please note,  the original title is now included as part of the commit message.

Reply:
okay will update

Reply:
this comment is not addressed yet. could you please reconsider it?

Reply:
should be done now